.github/workflows/ci.yml

```diff
@@ -72,6 +73,23 @@ jobs:
           name: pymgclient
           path: dist/
 
+  build_ubuntu_2204:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install system dependencies
+        run: |
+          sudo apt install -y libpython3.10 python3-pip python3-setuptools
+          sudo pip3 install networkx pytest pyopenssl sphinx
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Build source distribution
+        run: python3 setup.py sdist
+      - name: Install pymgclient
+        run: python3 -m pip install ./dist/pymgclient-*
+      - name: Import mgclient to validate installation
+        run: python3 -c "import mgclient"
+
```

Comment:
Once we have a package for Ubuntu 22.04 this job can be merged into  the `build_and_test_ubuntu` job.